client/packages/coldchain/src/Equipment/DetailView/UpdateStatusButton.tsx

```diff
@@ -145,7 +145,7 @@ export const UpdateStatusButtonComponent = ({
         nextButton={
           currentTab === Tabs.Status ? (
             <DialogButton
-              variant="next"
+              variant="next-and-ok"
```

Comment:
Renaming of the next button so I can create an actual 'next' button not 'ok & next'